src/controllers/organisationJobs.ts

```diff
@@ -0,0 +1,161 @@
+import moment from 'moment-timezone';
+import { CreateOrganisationJobSchema } from '../database/models';
+import { OrganisationJobsRepository, OrganisationsRepository } from '../database';
+import { asyncHandler } from '../middleware';
+import { checkOrganisationOwner, cleanseData, ErrorResponse } from '../utils';
+
+export class OrganisationJobsController {
+    constructor(
+        private readonly organisationJobsRepository: OrganisationJobsRepository,
+        private readonly organisationsRepository: OrganisationsRepository,
+    ) {
+        this.organisationJobsRepository = organisationJobsRepository;
+        this.organisationsRepository = organisationsRepository;
+    }
+
+    /**
+     * @desc    Get all jobs for a organisation
+     * @route   GET /api/organisations/:organisationId/jobs
+     * @access  Public
+     */
+    getOrganisationJobs = asyncHandler(async (req, res, next) => {
```

Comment:
Move the `asyncHandler` to wrap around the method in `organisationJobs.route.ts` instead

Reply:
Similarly for all the methods below too